lib/Myriad/Registry.pm

```diff
@@ -204,14 +175,17 @@ method add_emitter ($pkg, $method, $code, $args) {
     };
 }
 
-=head2 emitters_for
+=head2 emitter_for
```

Comment:
why did you change the name? it was more accurate

Reply:
I wanted to discuss that actually. I did it mainly to avoid irregular plurals. like `batches_for`
one thing also, for RPC we have it as `rpc_for` not `rpcs_for`

I do agree that `emitters_for` sounds way better to what it does as it returns a list of them.
However I do believe that their names should have some consistency. Maybe add `all` somewhere to the name?

Reply:
yes, `rpc` is a special case no need to make everything a special case :sweat_smile:

Reply:
`batch(es)` too